test/util/helpers.ts

```diff
@@ -111,7 +112,8 @@ export function buildMockParamsWithBlockParamAt(
  * param missing.
  *
  * @param index - The index within the `params` array where the "block" param
- * *would* appear.
+ * would* appear.
+ * @param blockParamIndex
```

Comment:
Is this left over from the JSDoc rules, since there is no description here?

Reply:
If `lint:fix` added it before you disabled that rule, then I think it makes sense to remove it, but if it's adding it even now then we can keep it. I was just curious.